app/src/main/java/org/fossasia/openevent/api/network/OpenEventAPI.java

```diff
@@ -17,21 +16,24 @@
  */
 public interface OpenEventAPI {
 
-    @GET("/speakers")
+
+    //TODO: Fix Hardcoding here
+    @GET("/event/2/speakers")
     void getSpeakers(Callback<SpeakerResponseList> speakerResponseListCallback);
 
-    @GET("/sponsors")
+    @GET("/event/2/sponsors")
     void getSponsors(Callback<SponsorResponseList> speakerResponseListCallback);
 
-    @GET("/sessions")
+    @GET("/event/2/sessions")
     void getSessions(Callback<SessionResponseList> sessionResponseListCallback);
 
-    @GET("/events")
+    @GET("/event")
     void getEvents(Callback<EventResponseList> eventResponseListCallback);
 
-    @GET("/tracks")
+    @GET("/event/2/microlocations")
```

Comment:
This is hardcoded to event number 2 right now. I'll have to think of a way to make it dynamic.